src/team-extension.ts

```diff
@@ -176,7 +183,9 @@ export class TeamExtension  {
     //Opens the file history page for the currently active file
     public OpenFileHistory(): void {
         if (this.ensureInitialized()) {
-            this._gitClient.OpenFileHistory(this._gitContext);
+            if (this._gitClient) {
+                this._gitClient.OpenFileHistory(this._repoContext);
+            }
```

Comment:
Seems like there should be an "else" on these commands that tells the user that we detected that this was not a git repository or something like that.